fuzz/README.md

```diff
@@ -0,0 +1,26 @@
+ # Fuzz Tests
```

Comment:
Maybe document that these have to be updated when adding fuzz tests?:
1. fuzz/Cargo.toml
2. fuzz/fuzz_targets/*.rs
3. CI